scss/components/_soe_navigation.scss

```diff
@@ -44,6 +44,12 @@
       margin-bottom: 6px;
     }
   }
+  
```

Comment:
@josephgknox, didn't you and @rebeccahongsf just have a conversation about your favorite text editor?

Does said text editor strip trailing whitespace on save?

Reply:
@jbickar we did, and my answer was Atom (though Sublime is a close second; just don't use it much anymore). I have a beautify plugin on both editors, but have them both set to not run on save. Rather, they need to be manually run. I do this to save for unnecessarily bloated PRs (in case configuration, organization, lint, etc. varies on projects). In either case, it could've saved me here.

Reply:
This is a nice little 1,2,3 of SWS: Joe is skillful, John is snarky, Shea is helpful.

Good talk, folks.

Reply:
Ah! Thank you.

![threestooges](https://user-images.githubusercontent.com/8933670/52080448-a8212f00-254c-11e9-9935-a1212da1bd63.jpg)